chiboycalix/andela-banka
# #165020592 Setup development environment

#### What does this PR do?
Setup development environment and install dependencies
#### Description of Task to be completed?
     - Download a good text Editor (VS code)
     - Run npm init to create a package.json file
     - Configure eslint to use the required style guide(airbnb)
     - Setup babel to compile your es6 code to es5
     - Setup test framework(mocha and chai)
     - Setup continious integration using travis ci and test coverage using code climate
#### How should this be manually tested?
     - By running npm start to see if the application is listening on the configured port
#### Any background context you want to provide?
     - None
#### What are the relevant pivotal tracker stories?
[#165020592] https://www.pivotaltracker.com/story/show/165020592
#### Screenshots (if appropriate)
#### Questions:
